administration/src/bp-modules/self-service/CardSelfServiceForm.tsx

```diff
@@ -0,0 +1,133 @@
+import ClearInputButton from '../../cards/extensions/components/ClearInputButton'
+import useWindowDimensions from '../../hooks/useWindowDimensions'
+import BasicDialog from '../../mui-modules/application/BasicDialog'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import { ExtensionForm } from '../cards/AddCardForm'
+import { ActionButton } from './components/ActionButton'
+import { IconTextButton } from './components/IconTextButton'
+
+const PrivacyButton = styled('button')`
+  border: none;
+  background-color: transparent;
+  color: blue;
+  text-decoration: underline;
+  padding: 0;
+  cursor: pointer;
+`
+
+const StyledCheckbox = styled(Checkbox)`
+  margin-top: 24px;
+  font-size: 16px;
+  margin-left: 4px;
+`
+
+const StyledAlert = styled(Alert)`
+  margin-bottom: 24px;
+  white-space: pre-line;
+`
+
+const Container = styled('div')`
+  margin-bottom: 24px;
+`
+
+type CardSelfServiceFormProps = {
+  card: CardBlueprint
+  notifyUpdate: () => void
+  dataPrivacyAccepted: boolean
+  setDataPrivacyAccepted: (value: boolean) => void
+  generateCards: () => Promise<void>
+}
+
+const getTooltipMessage = (cardsValid: boolean, dataPrivacyAccepted: boolean): string => {
+  const tooltipMessages: string[] = []
+  if (!cardsValid) {
+    tooltipMessages.push('Mindestens eine Ihrer Angaben ist ungültig.')
+  }
+  if (!dataPrivacyAccepted) {
+    tooltipMessages.push('Bitte akzeptieren Sie die Datenschutzerklärung.')
+  }
+
+  return tooltipMessages.join('\n')
+}
+const CardSelfServiceForm = ({
+  card,
+  notifyUpdate,
+  dataPrivacyAccepted,
+  setDataPrivacyAccepted,
+  generateCards,
+}: CardSelfServiceFormProps): ReactElement => {
+  const { viewportSmall } = useWindowDimensions()
+  const projectConfig = useContext(ProjectConfigContext)
+  const [openDataPrivacy, setOpenDataPrivacy] = useState<boolean>(false)
+  const [openReferenceInformation, setOpenReferenceInformation] = useState<boolean>(false)
+  const cardValid = card.isValid()
+  const cardCreationDisabled = !cardValid || !dataPrivacyAccepted
+  const clearNameInput = () => {
+    card.fullName = ''
+    notifyUpdate()
+  }
+
+  return (
+    <>
+      <Container key={card.id}>
+        <FormGroup label='Name'>
+          <InputGroup
+            large={viewportSmall}
+            placeholder='Erika Mustermann'
+            autoFocus
+            rightElement={
+              <ClearInputButton viewportSmall={viewportSmall} onClick={clearNameInput} input={card.fullName} />
+            }
+            intent={card.isFullNameValid() ? undefined : Intent.DANGER}
+            value={card.fullName}
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              card.fullName = event.target.value
+              notifyUpdate()
+            }}
+          />
+        </FormGroup>
+        {card.extensions.map((ext, i) => (
+          <ExtensionForm key={i} extension={ext} onUpdate={notifyUpdate} viewportSmall={viewportSmall} />
```

Comment:
hm okay. i guess we have to do that only for koblenz. since for nuernberg there are only cards for children under 16 i think @dkehne
I will add this task to #1670
Because if we restrict that we have to show a hint or the user will be confused